### ISSUE TITLE
Belching, excessive

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10036]
**Preferred term label:**
Belching, excessive

**Synonyms**
Eructation; burping

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0277999

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Auto-brewery syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000064
- label: Excessive eructation
- parents: [Constitutional symptom (HP:0025142)](https://hpo.jax.org/app/browse/term/HP:0025142)
- pmid: 
- def: Abnormally frequent belching (expelling gas noisily from the stomach through the mouth).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10036
- synonyms: Excessive burping(LAYPERSON_TERM)|Excessive belching(NONE)
